src/System Application/App/AI/src/Azure OpenAI/AOAIDeploymentsImpl.Codeunit.al

```diff
@@ -15,11 +16,15 @@ codeunit 7769 "AOAI Deployments Impl"
     InherentPermissions = X;
 
     var
+        Telemetry: Codeunit Telemetry;
         UnableToGetDeploymentNameErr: Label 'Unable to get deployment name, if this is a third party capability you must specify your own deployment name. You may need to contact your partner.';
         GPT4oLatestLbl: Label 'gpt-4o-latest', Locked = true;
         GPT4oPreviewLbl: Label 'gpt-4o-preview', Locked = true;
         GPT4oMiniLatestLbl: Label 'gpt-4o-mini-latest', Locked = true;
         GPT4oMiniPreviewLbl: Label 'gpt-4o-mini-preview', Locked = true;
+        DeprecatedDeployments: Dictionary of [Text, Date];
+        DeprecationDatesInitialized: Boolean;
+        DeprecationMessageLbl: Label 'Deployment %1 deprecated from %2. Check out codeunit 7768 AOAI Deployments', Comment = 'Telemetry message where %1 is the name of the deployment and %2 is the date of deprecation';
```

Comment:
Do we translate these strings or do we emit partner telemetry always in english?
If we translate them, you are good, but if we emit in english, you need to lock this string for translations, by adding `, Locked = true` after you close the quote of the comment